custom/ilsgateway/tests/handlers/utils.py

```diff
@@ -112,6 +114,7 @@ def tearDownClass(cls):
         SQLLocation.objects.all().delete()
         generator.delete_all_subscriptions()
         Domain.get_by_name(TEST_DOMAIN).delete()
+        super(TestScript, super(ILSTestScript, cls)).tearDownClass()
```

Comment:
some of the superclass methods do setup that conflicts with the subclass methods.  It's a sign that these tests are in need of serious refactoring, but I'm considering that out of scope for the django upgrade.

```super``` .... ```super``` (not surprisingly) didn't work, so I replaced it with https://github.com/dimagi/commcare-hq/pull/12135/commits/4d8c3cba1da378ff69dcb371de061f4adb3bd852